src/test/java/business/CommonSteps.java

```diff
@@ -0,0 +1,18 @@
+package business;
+
+import core.Config;
+import org.testng.annotations.Test;
+
+import static core.Config.logger;
+
+public class CommonSteps {
+    @Test
+    public void iInvokeLoginPageByUrl() {
+        logger.info("test starts");
+        logger.info("Init config file");
+        Config.initFile();
+        String url = Config.properties.getProperty("login.url");
+        System.out.println("I run test with url " + url);
```

Comment:
do not use System.out.println for any output. You have logger for that

Reply:
fixed